pykafka/handlers.py

```diff
@@ -86,6 +94,22 @@ def spawn(self, target, *args, **kwargs):
         return t
 
 
+class GEventHandler(Handler):
+    """A handler that uses a greenlet to perform its work"""
+    Queue = JoinableQueue
+    Event = GEEvent
+    Lock = GERLock  # fixme
```

Comment:
Oh, huh. Does it function the same as the RLock? If so, I guess it doesn't matter much? Worth testing to be sure it's compatible for how we're using the RLock.